fuzz/.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+[alias]
```

Comment:
Why does this file exist?

Reply:
I don't think this file buys enough to warrant its existence. Having aliases for `cargo install cargo-fuzz` doesn't seem useful to me. And aliasing `cargo fuzz_list` to `cargo fuzz list` seems a little weird? Like, why?